Bug: test_kernel_asic_mac_mismatch fails with IndexError: list index out of range

### Is it platform specific

generic

### Importance or Severity

### Description of the bug

arp/test_arp_update.py.test_kernel_asic_mac_mismatch fails with IndexError: list index out of range on t0-118

### Steps to Reproduce

Run arp/test_arp_update.py.test_kernel_asic_mac_mismatch on t0-118

### Actual Behavior and Expected Behavior

Test should pass

### Relevant log output

```shell
toggle_all_simulator_ports_to_rand_selected_tor = None
rand_selected_dut = <MultiAsicSonicHost>, ip_version = 6
setup_vlan_arp_responder = ('Vlan1000', IPv4Interface('192.169.0.1/22'), IPv6Interface('fc02:1000::1/64'), 40)

    @pytest.mark.parametrize("ip_version", [4, 6], ids=ip_version_string)
    def test_kernel_asic_mac_mismatch(
        toggle_all_simulator_ports_to_rand_selected_tor,  # noqa: F811
        rand_selected_dut, ip_version, setup_vlan_arp_responder,  # noqa: F811
        tbinfo
    ):
        vlan_name, ipv4_base, ipv6_base, ip_offset = setup_vlan_arp_responder
        if 'dualtor' in tbinfo['topo']['name']:
            servers = mux_cable_server_ip(rand_selected_dut)
            intf = random.choice(list(servers))
            if ip_version == 4:
                target_ip = servers[intf]['server_ipv4'].split('/')[0]
            else:
                target_ip = servers[intf]['server_ipv6'].split('/')[0]
        else:
            if ip_version == 4:
                target_ip = ipv4_base.ip + ip_offset
            else:
                target_ip = ipv6_base.ip + ip_offset
    
        rand_selected_dut.shell(f"ping -c1 -W1 {target_ip}; true")
    
        wait_until(10, 1, 0, neighbor_learned, rand_selected_dut, target_ip)
    
        neighbor_info = rand_selected_dut.shell(f"ip neigh show {target_ip}")["stdout"].split()
        pt_assert(neighbor_info[2] == vlan_name)
    
        asic_db_mac = rand_selected_dut.shell(
            f"sonic-db-cli APPL_DB hget 'NEIGH_TABLE:{vlan_name}:{target_ip}' 'neigh'"
        )['stdout']
>       pt_assert(neighbor_info[4].lower() == asic_db_mac.lower())
E       Failed: None
```

### Output of `show version`

```shell
202411
```

### Attach files (if any)

_No response_

--- a/tests/common/fixtures/ptfhost_utils.py
+++ b/tests/common/fixtures/ptfhost_utils.py
@@ -190,6 +190,8 @@ def setup_vlan_arp_responder(ptfhost, rand_selected_dut, tbinfo):
     for vlan, attrs in vlan_intf_config.items():
         for val in attrs:
             try:
+                if attrs[val].get('secondary') == 'true':
+                    continue
                 ip = ip_interface(val)
                 if ip.version == 4:
                     ipv4_base = ip

